backend_py/primary/primary/routers/parameters/router.py

```diff
@@ -8,93 +8,22 @@
 from webviz_services.sumo_access.parameter_types import EnsembleParameter, EnsembleSensitivity
```

Comment:
Unused imports (assume it will still be unused if you adjust according to previous comments)